config/locales/en.yml

```diff
@@ -45,6 +45,14 @@ en:
     users:
       create:
         error: You must provide a users email address e.g. bin/rails users:create EMAIL_ADDRESS=name@example.com
-  project_list:
-    title:
-      all: All projects
+  project:
```

Comment:
We can leverage the GOV.UK Form Builder locales here by following convention here:

https://govuk-form-builder.netlify.app/building-blocks/localisation/#populating-label-caption-and-hint-text-from-the-localisation-data